test.js

```diff
@@ -3,20 +3,24 @@ import difflet from 'difflet';
 import m from './';
 
 const diff = difflet({indent: 2});
-
-test.cb(t => {
-	const expected = {
-		'elements/colors': {
-			style: true,
-			documentation: true
-		},
-		'elements/grid': {
-			style: true,
-			documentation: true
+const expected = {
+	elements: {
+		colors: {
+			path: 'src/elements/colors/',
+			style: 'styles.scss',
```

Comment:
nah I would let this as a convention for now.
we only include the default styles.scss.

If you want more files you can but use _file1.scss _file2.scss and included them in styles